### PR TITLE
Add exercise preview component 

### PR DESCRIPTION
**Pull request by nathanstitt**
So it can be used in exercises

TODO:
- [x] Add spec
- [x] add it to the testing/preview page
- [x] Maybe have the component calculate it's own LO/Content vs using the awkward prop function
- [x] Remove component from Tutor and use this
- [x] Fix preview styles with the slightly modified mark up
- [x] Confirm component functionality in `exercise-js`

### CODE REVIEW

**Comment by jivewise** [comment]
I updated the order of the stem and stimulus per Dante's request here: https://github.com/openstax/exercises-js/issues/48

File: resources/styles/components/exercise/preview.less [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+  }
+
+
+  &.is-displaying-feedback {
+    .panel-body .answers-table .question-feedback-content { display: table; }
+    .detailed-solution{ display: block; }
+  }
+
+  // use absolute positioning for the identifer so it's link is clickable through the overlay
+  .panel-footer .controls {
```
Comment (nathanstitt) [inline resources/styles/components/exercise/preview.less:176]: comment is not accurate, it's no longer absolutly positioned

**Comment by nathanstitt** [comment]
Looks good to me, other than the tiny incorrect comment I noted.

@jivewise, you mentioned something about the preview not working correctly on exercises, but it all appears fine to me.  Can you refresh my memory on what was the issue ?

**Comment by jivewise** [comment]
I believe we just need to make sure that both the Detailed Solution and Answer feedback for each answer is showing up in the preview.

**Pull request merged**